algorithms/yiddish/stemmer.tt

```diff
@@ -0,0 +1,215 @@
+    <li>Stressed: אױס, אױפ, אומ, אונטער, איבער, אײֲנ, אָנ, אָפּ, בײַ, פאָר, צו.</li>
+    <li>Unstressed: אַנט, באַ, דער, צע</li>
+  </ul>
+  <li>If the verbal prefix is followed by גע, it is replaced with "GE" and the cursor is advanced (e.g. אַװעקגעגאַנגען).</li>
+  <li>If the verbal prefix is followed by צו, it is replaced with "TSU" and the cursor is advanced (e.g. אַרומצוגײן).</li>
+</ul>
+
+<p>We are now at the start of the main portion of the word (past any verbal prefix and past participle marker).</p>
+
+<ul>
+  <li>The following valid Yiddish three-consonant sequences are skipped: שפר, שטר, שטש, דזש.</li>
+  <li>If there is a sequence of three consonants, the cursor is advanced past them, and P1 is marked.</li>
+  <li>Otherwise, the cursor is advanced to the first vowel, and then up to the first non-vowel, minus 1, and P1 is marked.</li>
+  <li>If P1 is not at least 3 letters beyond the main portion, it is advanced past the 3rd letter.</li>
+</ul>
+
+<h3>Backwards mode</h3>
+
+<p>Unless otherwise stated, all deletes ensure we are beyond P1.</p>
+<p>In each pass, at the first level of bullets, the longest matching suffix always wins.</p>
+
+<p>First pass:</p>
+<ul>
+  <li>Delete plural/adjective endings: ער, ערס, ן, ס, ען, נס, ענערס, ענס, עס.</li>
+  <ul>
+    <li>Exceptions: יע is not deleted (e.g. אגיטאַציע), יעס becomes יע</li>
+  </ul>
+  <li>Delete plural/adjective endings: ענעם, ענער, ענע, ענס</li>
+  <ul>
+    <li>If preceded by an irregular past participle ending in ן, replace it with the stem, e.g. געגאַנגען becomes גײ.</li>
+  </ul>
+  <li>Delete the verb/past participle ending: ט.</li>
+  <ul>
+    <li>Because of the above, also delete noun/adjectives ending in טן, טע, טער, טס so that they stem identically to the equivalent noun with the ט.</li>
+    <li>Similarly for past participles: -tns, -tene, -tenem, -tener טנס, טענע, טענעם, טענער</li>
+    <li>If the ט was before P1, we try to perform the same action while leaving the ט in place</li>
+    <li>At the same time, if preceded by an irregular past participles ending in ט, replace it with the stem, e.g. געבראַכט becomes ברענג.</li>
+  </ul>
+  <li>Anything ending with גײן is transformed to גײ.</li>
+  <li>Anything ending in an irregular past participle is corrected to the stem.</li>
+  <li>Delete noun endings: ונג, הײט, קײט, יקײט, שאַפֿט.</li>
+  <li>Delete noun endings: יזם, יזמס.</li>
+  <li>Delete Hebraic plural ending: ים</li>
+  <li>Replace the Hebraic plural ending ות with ה.</li>
+  <li>Delete the diminutive endings: עלעך, עלע, לעך, עלעס, עלען.</li>
+  <li>Delete the noun endings: יסט, יסטן.</li>
+  <ul>
+    <li>Exceptions to the above: words ending in גיסט or שיסט.</li>
+  </ul>
+  <li>Delete the verb ending סטו.</li>
+  <li>Delete the uperlative endings: סטער, סטע, סטן.</li>
```

Comment:
I just noticed "uperlative" when checking the generated website - is that a typo for "superlative"?

Reply:
Yes, definitely a typo for "superlative"! I'll add a new pull request for this, and also do a full reread of the text and code comments.

Reply:
Thanks for confirming - no need for a PR, I've now pushed a fix.